src/apps/explorer/components/OrderWidget/index.tsx

```diff
@@ -1,9 +1,30 @@
-import React from 'react'
-
+import React, { useEffect, useState } from 'react'
 import { useParams } from 'react-router'
 
+import { getOrder, RawOrder } from 'api/operator'
+
 export const OrderWidget: React.FC = () => {
+  const [order, setOrder] = useState<RawOrder | null>(null)
+  const [error, setError] = useState('')
+
   const { orderId } = useParams<{ orderId: string }>()
 
-  return <div>You are viewing order id: {orderId}</div>
+  useEffect(() => {
+    // TODO: get network from a hook or something
+    getOrder({ networkId: 4, orderId })
+      .then((order) => {
+        console.log('got order', order)
```

Comment:
just for testing right? maybe a TODO to remove in case u forget

Reply:
Yep yep, all of this will be rewritten